.github/ISSUE_TEMPLATE/vaos-bug.md

```diff
@@ -32,7 +32,5 @@ _Describe what was supposed to happen; reference existing Github issue that cont
 
 ---
 ## How to configure this issue
-- [ ] **Labeled with Team** (`appointments`)
```

Comment:
`appointments` and `bug` are already applied by default so no need to add it as a step here.